src/ol/observable.js

```diff
@@ -62,6 +62,18 @@ ol.Observable.prototype.changed = function() {
 
 
 /**
+ * Dispatches an event (or event like object) and calls all listeners
+ * listening for events of this type. The type of the event is decided by the
+ * type property on the event object.
+ *
+ * @param {goog.events.EventLike} event Event object.
+ * @function
+ * @api
+ */
+ol.Observable.prototype.dispatchEvent;
```

Comment:
Ok maybe it works, I see that we have the same thing for `ol.format.GeoJSON.prototype.readFeature` [for example](https://github.com/openlayers/ol3/blob/master/src/ol/format/geojsonformat.js#L384).